app/controllers/plans_controller.rb

```diff
@@ -33,15 +33,14 @@ def get_started
     if request.post? && request.xhr?
       if @get_started_form.valid?
         url =
-          plan_goals_url(
-            country_name: @get_started_form.country.name,
-            assessment_type: @get_started_form.assessment_type,
-            plan_term: @get_started_form.plan_term_s,
-            areas: @get_started_form.technical_area_ids.join("-"),
-          )
-        Rails.logger.info "Redirect workaround for an XHR request to URL: #{
-                            url
-                          }"
+            plan_goals_url(
+                country_name: @get_started_form.country.name,
+                assessment_type: @get_started_form.assessment_type,
+                plan_term: @get_started_form.plan_term_s,
+                areas: @get_started_form.technical_area_ids.blank? ? nil : @get_started_form.technical_area_ids.join("-"),
```

Comment:
Let see this phrased in positive terms: `@get_started_form.technical_area_ids.present?`

Reply:
fixed